src/modules/mavlink/mavlink_mission.cpp

```diff
@@ -502,10 +524,9 @@ MavlinkMissionManager::send()
 		}
 
 	} else if (_slow_rate_limiter.check(hrt_absolute_time())) {
-		if ((_count[MAV_MISSION_TYPE_MISSION] > 0) && (_current_seq >= 0)) {
-
-			send_mission_current(_current_seq);
+		send_mission_current(_current_seq);
```

Comment:
Why is this needed?

Reply:
The mission current has now the IDs which should be periodically send. That means it should send it periodically as well, when there is no mission available, so that the ground station is informed when the mission is cleared.